docs/docs/configuration/cache.md

```diff
@@ -9,24 +9,23 @@ The cache directory includes
 The cache option is common to all scanners.
 
 ## Clear Caches
-The `--clear-cache` option removes caches.
-
-**The scan is not performed.**
+`trivy clean` subcommand removes caches.
 
 ```
-$ trivy image --clear-cache
+$ trivy clean --scan-cache
 ```
 
 <details>
 <summary>Result</summary>
 
 ```
-2019-11-15T15:13:26.209+0200    INFO    Reopening vulnerability DB
-2019-11-15T15:13:26.209+0200    INFO    Removing image caches...
+2024-06-21T21:58:21+04:00       INFO    Removing vulnerability database...
```

Comment:
```suggestion
2024-06-21T21:58:21+04:00       INFO    Removing scan cache...
```

Reply:
If you will use `-all` flag.
```suggestion
2024-06-21T21:58:21+04:00       INFO    Removing all caches...
```

Reply:
Fixed in https://github.com/aquasecurity/trivy/pull/6993/commits/361db194a15948ccd9a94b676d85b17431ca168b